staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -566,7 +575,10 @@ See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts`
 
 		if o.DryRunStrategy != cmdutil.DryRunClient {
 			// Then create the resource and skip the three-way merge
-			obj, err := helper.Create(info.Namespace, true, info.Object)
+			createOptions := &metav1.CreateOptions{
+				FieldValidation: o.ValidationDirective,
+			}
+			obj, err := helper.CreateWithOptions(info.Namespace, true, info.Object, createOptions)
```

Comment:
Why does the helper need the validation directive twice?

Reply:
ya, you're totally right, this is unecessary now that we've added `WithFieldValidation` to the helper. I didn't realize that we're using the helper here (and elsewhere). I've reverted all of it in the most recent commit.